pulsar-functions/localrun/src/main/java/org/apache/pulsar/functions/LocalRunner.java

```diff
@@ -569,15 +582,20 @@ private void startThreadedMode(org.apache.pulsar.functions.proto.Function.Functi
             instanceConfig.setFunctionId(UUID.randomUUID().toString());
             instanceConfig.setInstanceId(i + instanceIdOffset);
             instanceConfig.setMaxBufferedTuples(1024);
-            instanceConfig.setPort(FunctionCommon.findAvailablePort());
```

Comment:
What about this?

Reply:
There is no point in setting any of these ports.  They are not used when running in threaded mode for local run